### PR TITLE
Improvements to the Qt5.15 sample application for developers.

### PR DESCRIPTION
**Pull request by piggz**
* Install in /usr
* Dont use a resource file, use normal Sailfish qml paths

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -19,4 +57,4 @@ int main(int argc, char *argv[])
     engine.load(url);
 
     return app.exec();
-}
+};
```
Comment (rinigus) [inline src/main.cpp]: Don't think we need `;`

**Review by rinigus — APPROVED** [review]
Very lambda-intense, but looks to be fine. I have nothing against qrc files, quite handy in this respect. But this works as well.

File: src/main.cpp (hunk 1 above)
Comment (piggz) [reply src/main.cpp]: true!

**Comment by piggz** [comment]
> Very lambda-intense, but looks to be fine. I have nothing against qrc files, quite handy in this respect. But this works as well.

Yes, i just thought to keep it like a sailfish app for other devs

**Pull request merged**